src/tool/cpp/cppwinrt/component_writers.h

```diff
@@ -198,7 +198,11 @@ int32_t WINRT_CALL WINRT_GetActivationFactory(void* classId, void** factory) noe
 
         w.write(format,
             bind_each<write_component_include>(classes),
-            bind_each<write_component_activation>(classes));
+            settings.component_lib,
```

Comment:
Out of curiosity, why is this line repeated? Or am I just not seeing the difference?

Reply:
You mean the `settings.component_lib` argument? Because the same value is needed multiple times by the format string.